packages/core/src/test/amazonqFeatureDev/session/sessionState.test.ts

```diff
@@ -27,10 +27,10 @@ const mockSessionStateAction = (msg?: string): SessionStateAction => {
             new AppToWebViewMessageDispatcher(new MessagePublisher<any>(new vscode.EventEmitter<any>()))
         ),
         telemetry: new TelemetryHelper(),
+        uploadHistory: {},
```

Comment:
nit: Consider adding a test where uploadHistory is populated and show we can access an element from it

Reply:
Sorry, used the wrong account here.

Reply:
Oh, I thought was somekind of automatic messaging. Thanks, yea, due the timeline I would say we can add those testing scenarios as a follow up